src/core/Modal/Modal/__snapshots__/Modal.test.tsx.snap

```diff
@@ -641,7 +641,7 @@ exports[`Basic modal should match snapshot 1`] = `
 }
 
 .c8.fi-button--secondary:hover {
-  background: linear-gradient(0deg,hsl(202,7%,93%) 0%,hsl(0,0%,100%) 100%);
+  background: linear-gradient(-180deg,#ecedee 0%,rgba(255,255,255,0) 100%);
```

Comment:
I assume this hsl/hex conversion comes from using alphaHex?

Reply:
Seems to be. Only tokens are used in the code.